po/fr.po

```diff
@@ -0,0 +1,296 @@
+msgid "New tab"
+msgstr "Nouvel onglet"
+
+#: shortcuts-dialog.blp:13
+msgid "New window"
+msgstr "Nouvelle fenêtre"
+
+#: shortcuts-dialog.blp:67
+msgid "Next tab"
+msgstr "Onglet suivant"
+
+#: Window.cpp:236 Tab.cpp:194
+msgid "OK"
+msgstr "OK"
+
+#: shortcuts-dialog.blp:21
+msgid "Open file"
+msgstr "Ouvrir un fichier"
+
+#. Translators: this is a menu item for pinning a tab.
+#: window.blp:26
+msgid "P_in"
+msgstr "_Épingler"
+
+#: shortcuts-dialog.blp:71
+msgid "Previous tab"
+msgstr "Onglet précédent"
+
+#: shortcuts-dialog.blp:29
+msgid "Quit"
+msgstr "Quitter"
+
+#: window.blp:169 window.blp:294
+msgid "Reload Page"
+msgstr "Recharger la Page"
+
+#: shortcuts-dialog.blp:46
+msgid "Reload current page"
+msgstr "Actualiser"
+
+#: shortcuts-dialog.blp:89
+msgid "Reset zoom"
+msgstr "Taille réelle"
+
+#: window.blp:236
+msgid "Restore Zoom"
+msgstr "Taille réelle"
+
+#: Application.cpp:49
+msgid "Sam Atkins"
+msgstr "Sam Atkins"
+
+#. Translators: this is the placeholder text for the main location entry.
+#: window.blp:188
+msgid "Search or enter a URL to open…"
+msgstr "Rechercher ou saisir une adresse…"
+
+#: Application.cpp:57
+msgid "SerenityOS developers"
+msgstr "Développeurs SerenityOS"
+
+#: Application.cpp:70
+msgid "SerenityOS website"
+msgstr "Site web de SerenityOS"
+
+#. Translators: Сергей Бугаев in Cyrillic
+#: Application.cpp:44
+msgid "Sergey Bugaev"
+msgstr "Sergey Bugaev"
+
+#: shortcuts-dialog.blp:8
+msgid "Shortcuts"
+msgstr "Raccourcis"
+
+#: shortcuts-dialog.blp:25
+msgid "Show shortcuts"
+msgstr "Afficher les raccourcis"
+
+#: window.blp:212
+msgid "Tab Overview"
+msgstr "Aperçu de l‘Onglet"
+
+#: shortcuts-dialog.blp:56
+msgid "Tabs"
+msgstr "Onglets"
+
+#. Translators: this is a menu item for unpinning a tab.
+#: window.blp:32
+msgid "Unp_in"
+msgstr "Désépingler"
+
+#: Tab.cpp:192
+msgid "Web page alerts:"
+msgstr "Message de la page:"
+
+#: Tab.cpp:237
+msgid "Web page asks for input:"
+msgstr ":"
+
+#: Tab.cpp:213
+msgid "Web page asks you to confirm:"
+msgstr "Webbsidan ber dig bekräfta:"
+
+#: shortcuts-dialog.blp:77
+msgid "Zoom"
+msgstr "Zoom"
+
+#: window.blp:246
+msgid "Zoom In"
+msgstr "Zooma avant"
+
+#: window.blp:231
+msgid "Zoom Out"
+msgstr "Zooma arrière"
+
+#: shortcuts-dialog.blp:80
+msgid "Zoom in"
+msgstr "Zoom avant"
+
+#: shortcuts-dialog.blp:85
+msgid "Zoom out"
+msgstr "Zooma arrière"
+
+#: window.blp:18 window.blp:46
+msgid "_About Ladybird"
+msgstr "_À propos de Ladybird"
+
+#: window.blp:38
+msgid "_Close"
+msgstr "_Fermer"
+
+#: window.blp:23
+msgid "_Duplicate"
+msgstr "_Dupliquer"
+
+#: window.blp:13 window.blp:45
+msgid "_Keyboard Shortcuts"
+msgstr "_Raccourcis Clavier"
+
+#: window.blp:36
+msgid "_Move to New Window"
+msgstr "_Déplacer vers une nouvelle fenêtre"
+
+#: window.blp:10 window.blp:42
+msgid "_New Window"
+msgstr "_Nouvelle Fenêtre"
+
+#: window.blp:12 window.blp:44
+msgid "_Open File"
+msgstr "_Ouvrir un Fichier"
+
+#: window.blp:22
+msgid "_Reload"
+msgstr "_Recharger"
```

Comment:
Above, you translated "Reload current page" as Actualiser; is this intentional?

Reply:
It was intentional because again, that's what is used in most browsers and what I thought was the best here.

Reply:
OK then 🙂